Common++/header/IpAddress.h

```diff
@@ -41,19 +40,27 @@ namespace pcpp
 		 * A constructor that creates an instance of the class out of 4-byte integer value.
 		 * @param[in] addrAsInt The address as 4-byte integer in network byte order
 		 */
-		IPv4Address(const uint32_t addrAsInt) { memcpy(m_Bytes.data(), &addrAsInt, sizeof(addrAsInt)); }
+		IPv4Address(const uint32_t addrAsInt)
+		{
+			memcpy(m_Bytes.data(), &addrAsInt, sizeof(addrAsInt));
+		}
 
 		/**
 		 * A constructor that creates an instance of the class out of 4-byte array.
 		 * @param[in] bytes The address as 4-byte array in network byte order
 		 */
-		IPv4Address(const uint8_t bytes[4]) { memcpy(m_Bytes.data(), bytes, 4 * sizeof(uint8_t)); }
+		IPv4Address(const uint8_t bytes[4])
+		{
+			memcpy(m_Bytes.data(), bytes, 4 * sizeof(uint8_t));
+		}
 
 		/**
 		 * A constructor that creates an instance of the class out of a 4-byte standard array.
 		 * @param[in] bytes The address as 4-byte standard array in network byte order
 		 */
-		IPv4Address(const std::array<uint8_t, 4>& bytes) : m_Bytes(bytes) {}
+		IPv4Address(const std::array<uint8_t, 4>& bytes) : m_Bytes(bytes)
+		{
+		}
```

Comment:
This formatting change doesn't make a lot of sense, is there an option to avoid it?

Reply:
this one is due to `BreakBeforeBraces: Allman`

Reply:
If we change to `Custom` we can set `SplitEmptyFunction: false` which I think should solve it:
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#bracewrapping

Reply:
actually, I would stand for splitting the braces in this case. shall we just use the default `Allman` settings to prevent too much customization from using `Custom`?

Reply:
I don't know the default values of the different flags under `BraceWrapping`. If all the defaults are what we need, I think it's ok to set:
```
BreakBeforeBraces: Custom
BraceWrapping:
  SplitEmptyFunction: false
```

Reply:
I tried your config, and the effect is like this:

```
void matchNetwork(const std::string& network) {}
```
become this:
```
void matchNetwork(const std::string& network)
{}
```

Reply:
this is slightly better I think. The rest of the curly braces formatting stays the same?

Reply:
if there is something in the braces, yes, it will be the same.
```
void matchNetwork(const std::string& network)
{
   // ...
}
```

Reply:
sure, applied your config.